Jenkinsfile

```diff
@@ -2,4 +2,4 @@
  See the documentation for more options:
  https://github.com/jenkins-infra/pipeline-library/
 */
-buildPlugin(useAci: true)
+buildPlugin(useContainerAgent: true)
```

Comment:
useAci is deprecated: https://github.com/jenkinsci/jenkins-infra-test-plugin/pull/12/commits/c6d681337dfcf14a811a5e2d44b325ebe2bc86a2